### DISCUSSION TITLE
Feat: De-dupe and show info when adding the same trace/observation to an annotation queue multiple times

### DISCUSSION THREAD

**Discussion by marliessophie** [langfuse/langfuse#4035] [Ideas]
### Describe the feature or potential improvement

User feedback: 
"It would be helpful to either automatically remove duplicates from the queue or display a warning when duplicate Traces are added."

**Comment by marcklingen** [comment]
Agree, I'd suggest to default to automatically deduping items when they are added to the annotation queue
